rancher-common/kubernetes.tf

```diff
@@ -1,62 +1,9 @@
 # Kubernetes resources
 
-# Create cert-manager-crd service account
-resource "kubernetes_service_account" "cert_manager_crd" {
-  depends_on = [rke_cluster.rancher_cluster]
-
-  metadata {
-    name      = "cert-manager-crd"
-    namespace = "kube-system"
-  }
-
-  automount_service_account_token = true
-}
-
-# Bind cert-manager-crd service account to cluster-admin
-resource "kubernetes_cluster_role_binding" "cert_manager_crd_admin" {
-  depends_on = [rke_cluster.rancher_cluster]
-
-  metadata {
-    name = "${kubernetes_service_account.cert_manager_crd.metadata[0].name}-admin"
-  }
-  role_ref {
-    api_group = "rbac.authorization.k8s.io"
-    kind      = "ClusterRole"
-    name      = "cluster-admin"
-  }
-  subject {
-    kind      = "ServiceAccount"
-    name      = kubernetes_service_account.cert_manager_crd.metadata[0].name
-    namespace = "kube-system"
-  }
-}
-
-locals {
-  # split_rke_kubernetes_version
-  srkv          = split("-", var.rke_kubernetes_version)
-  hyperkube_tag = join("-", [local.srkv[0], local.srkv[1]])
+data "http" "cert_manager_crds" {
+  url = "https://raw.githubusercontent.com/jetstack/cert-manager/release-${regex("([0-9]+\\.[0-9]+)", var.cert_manager_version)[0]}/deploy/manifests/00-crds.yaml"
```

Comment:
Looks like the suggested CRD manifest URL from the past 3 minor releases of cert-manager can use the version variable directly, could this be simpler?

Here's what I pulled from the installation instructions:
- `https://github.com/jetstack/cert-manager/releases/download/v0.14.3/cert-manager.crds.yaml`
- `https://github.com/jetstack/cert-manager/releases/download/v0.15.1/cert-manager.crds.yaml`
- `https://github.com/jetstack/cert-manager/releases/download/v0.16.1/cert-manager.crds.yaml`

Reply:
I did not want to change the cert-manager version in this PR to not make it bigger and for 0.12.0 this does not seem to be available. But when we upgrade, we can change the URL.

Reply:
Oh right, forgot we hadn't merged that upgrade in yet - thank you!